content/docs/2.10/scalers/azure-app-insights.md

```diff
@@ -30,6 +30,7 @@ triggers:
     appInsightsResourceURL: https://api.applicationinsights.airgap.io/
     # Required when cloud = Private.
     activeDirectoryEndpoint: https://login.airgap.example/
+    ignoreNullValues: false # Default is `true`, which means ignoring any errors with data extraction from a successful query. Set to `false` the scaler will return error when null values are discovered 
```

Comment:
This changes the current behavior (now we raise an error), we should maintain current behaviour

Reply:
Done